src/controllers/itemdetailpage.js

```diff
@@ -589,242 +1098,394 @@ define(["loading", "appRouter", "layoutManager", "connectionManager", "cardBuild
+
             similarCollapsible.classList.remove("hide");
-            var apiClient = connectionManager.getApiClient(item.ServerId),
-                options = {
-                    userId: apiClient.getCurrentUserId(),
-                    limit: 12,
-                    fields: "PrimaryImageAspectRatio,UserData,CanDelete"
-                };
-            "MusicAlbum" == item.Type && item.AlbumArtists && item.AlbumArtists.length && (options.ExcludeArtistIds = item.AlbumArtists[0].Id), apiClient.getSimilarItems(item.Id, options).then(function(result) {
-                if (!result.Items.length) return void similarCollapsible.classList.add("hide");
+            var apiClient = connectionManager.getApiClient(item.ServerId);
+            var options = {
+                userId: apiClient.getCurrentUserId(),
+                limit: 12,
+                fields: "PrimaryImageAspectRatio,UserData,CanDelete"
+            };
+
+            if ("MusicAlbum" == item.Type && item.AlbumArtists && item.AlbumArtists.length) {
+                options.ExcludeArtistIds = item.AlbumArtists[0].Id;
+            }
+
+            apiClient.getSimilarItems(item.Id, options).then(function (result) {
+                if (!result.Items.length) {
+                    return void similarCollapsible.classList.add("hide");
+                }
+
                 similarCollapsible.classList.remove("hide");
                 var html = "";
                 html += cardBuilder.getCardsHtml({
                     items: result.Items,
                     shape: "autooverflow",
                     showParentTitle: "MusicAlbum" == item.Type,
-                    centerText: !0,
-                    showTitle: !0,
+                    centerText: true,
+                    showTitle: true,
                     context: context,
-                    lazy: !0,
-                    showDetailsMenu: !0,
+                    lazy: true,
+                    showDetailsMenu: true,
                     coverImage: "MusicAlbum" == item.Type || "MusicArtist" == item.Type,
-                    overlayPlayButton: !0,
-                    overlayText: !1,
+                    overlayPlayButton: true,
+                    overlayText: false,
                     showYear: "Movie" === item.Type || "Trailer" === item.Type
                 });
                 var similarContent = similarCollapsible.querySelector(".similarContent");
-                similarContent.innerHTML = html, imageLoader.lazyChildren(similarContent)
-            })
+                similarContent.innerHTML = html;
+                imageLoader.lazyChildren(similarContent);
+            });
         }
     }
 
     function renderSeriesAirTime(page, item, isStatic) {
         var seriesAirTime = page.querySelector("#seriesAirTime");
-        if ("Series" != item.Type) return void seriesAirTime.classList.add("hide");
+        if ("Series" != item.Type) {
+            return void seriesAirTime.classList.add("hide");
```

Comment:
```suggestion
            seriesAirTime.classList.add("hide");
            return;
```
sorry, missed that when was composing my suggestion

Reply:
https://github.com/jellyfin/jellyfin-web/pull/509/commits/47cc029c7056b2ad334a93ba80f1568c6020a0ad